src/lib/mediaUtil.js

```diff
@@ -1,3 +1,36 @@
+// import { notEmptyString } from './formValidators';
+import { lookupReadableMetadataName } from './explorerUtil';
+
 export const VIEW_ALL_STORIES = 'VIEW_ALL_STORIES';
 export const VIEW_REGULARLY_COLLECTED = 'VIEW_REGULARLY_COLLECTED';
 export const ALL_MEDIA = -1;
+
+/* this is set up to match the expected params within the metadataCheckboxFieldArray */
+export function importTagsFromCustomSearches(tagsObj) {
+  if (tagsObj) {
+    const updatedTagsObj = { tags: {} };
```

Comment:
Does this need a `tags` field? It looks like it could be initialized as an empty object and referenced directly instead of using `tags`.

Reply:
true!